### PR TITLE
first pass at creating a separate section for calibnet

### PR DESCRIPTION
**Pull request by walkerlj0**
Hey, here is what I was thinking I could add to your Tutorial- A second option for setting up on calibnet. All I have done so far is make an exact copy of the one you have, I still need to put in the new instructions for building from source, with a link to the right binary, the correct glif node for calibnet to connect to, and the instructions to make sure you are installing the right version of the binary.

What do you think? If this layout for the tut works for you, I can put in the rest of the info for the calibnet page, then re-submit this PR for review

### CODE REVIEW

**Comment by LexLuthr** [comment]
@walkerlj0 It looks good. Please complete the PR and TSE team can review it. There might be some missing details in tutorials so I am hoping you can identify the gaps and fix those. Let me know if you need any help.

**Comment by longfeiWan9** [comment]
@walkerlj0 I think maybe we can have one page for `setup` with two sections, 1 for mainnet and another for calibnet because I see many duplicated or repeated information between these two pages.

File: content/en/tutorials/lotus/store-and-retrieve/set-up-calibnet.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+Programs that interact with the Filecoin network are called _implementations_, and Lotus is a command-line interface (CLI) implementation. There are other implementation being created alongside Lotus, however Lotus is the only Filecoin implementation created and maintained by Protocol Labs.
+{{< /alert >}}
+
+### Things to note
+
+As you're going through this section, make a note of the following variables:
+
+| Variable | Description | Example |
+| --- | --- | --- |
+| Your Filecoin address | The public part of your Filecoin address. This address is what other users can use to send your FIL. | `f1fwavjcfb32nxbczmh3kgdxhbffqjfsfby2otloi` |
```
Comment (longfeiWan9) [inline content/en/tutorials/lotus/store-and-retrieve/set-up-calibnet.md:30]: testnet wallet address starts with t.

**Comment by walkerlj0** [comment]
@longfeiWan9 in our session today where people were trying to set these up, there was a lot of confusion about .. well, the fact that these are different (mainnet and testnet) and I htink if we try and put all of it on one page/ in one workflow, it will
1) Be hard to understand that these are two different options for two different 'nets
2) Know which steps you do and which ones you skip for each respective mainet/ testnet. 


I don't think putting these things on the same page will give enough of a clear workflow or convey the fact that these are two separate chains that you can join.

We have two options if we were to put this on the same page:
In each step, give an option to Do A if you are on mainnet, and do B if you are on Calibenet. 
This will be incredibly  confusing.
 

If you simply take a second workflow for calibnet, with repeated steps after the mainnet setup, I don't think, well, anyone will take the choice to setup on mainnet.

I honestly think giving two different options in the menu is a very important thing, content-wise, to present.

**Comment by SgtCoin** [comment]
On a bone stock Ubuntu build, the following requirements are needed to compile lotus.

- jq
- make
- cargo
- mesa-opencl-icd
- ocl-icd-opencl-dev
- hwloc libhwloc-dev

Should those be listed in the Prerequisites section?

Also, the git information didnt work. `git clone https://github.com/filecoin-project/lotus/<releases/version>`
I had to use https://github.com/filecoin-project/lotus, cd to lotus and checkout v1.16.1. 

Once lotus installs, you need to add replace `lotus `with `./lotus` in the command. Once it starts running, you will get the following error over and over again. 

`2022-07-18T20:50:49.809Z        ERROR   events  events/observer.go:58   listen head changes errored: listenHeadChanges ChainNotify call failed: RPC error (-32601): method 'Filecoin.ChainNotify' not supported in this mode (no out channel support)
2022-07-18T20:50:50.811Z        INFO    events  events/observer.go:69   restarting listenHeadChanges
`
You can try to fix this by changing the endpoint to calibnet. 

`FULLNODE_API_INFO=https://calibration.node.glif.io/ ./lotus daemon --lite`

It doesnt seem to work either. 

`2022-07-18T20:56:03.625Z        ERROR   events  events/observer.go:58   listen head changes errored: listenHeadChanges ChainNotify call failed: RPC client error: sendRequest failed: http status 503 Service Unavailable unmarshaling response: EOF
2022-07-18T20:56:04.627Z        INFO    events  events/observer.go:69   restarting listenHeadChanges
`

**Comment by rjan90** [comment]
Closing this PR as outdated

**Pull request closed without merging**